safety_limiter/src/safety_limiter.cpp

```diff
@@ -617,6 +626,15 @@ class SafetyLimiterNode
       last_disable_cmd_ = ros::Time::now();
     }
   }
+
+  void diagnoseCollision(diagnostic_updater::DiagnosticStatusWrapper &stat)
+  {
+    if (col_points_.points.size() > 0)
+      stat.summary(diagnostic_msgs::DiagnosticStatus::WARN, "Collision points exist inside of the footprint.");
+    else
+      stat.summary(diagnostic_msgs::DiagnosticStatus::OK, "No collision point exists.");
+    stat.addf("Collision points", "%u", col_points_.points.size());
```

Comment:
Although it does not directly relate to this PR, I found that `col_points_.points.size()` is always 14 (or 0 if there are no colliding points) in case of our robot. The position of the point seems to be correct, but all the same. Is it intended ?

<details>

```
$rostopic echo /collision

header: 
  seq: 11259
  stamp: 
    secs: 1538643813
    nsecs: 588606292
  frame_id: "base_link"
points: 
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
  - 
    x: 0.0655297264457
    y: -0.227409601212
    z: 0.0
channels: []
---
```

</details>

Reply:
@ykoga-kyutech This happens if the robot is in collision at beginning and takes at least 14 timesteps to escape from collision.
In this state, it has two cases:
- the robot can escape from collision (`r_lim_ > 0`)
- the robot can't escape from collision (`r_lim_ == 0`)

If the robot in not in collision state at beginning, number of the points is always 1 or 0.

The number of the collision points has less meaning.
It's better to output `r_lim_` (velocity limit ratio.)

Reply:
Thx for the detailed explanation! I'll fix and try it on my robot soon.